config/config.go

```diff
@@ -265,6 +265,27 @@ type DeploymentConfig struct {
 // defaults to the namespace configured for IstioNamespace (which itself defaults to 'istio-system').
 type IstioComponentNamespaces map[string]string
 
+
+// Threshold configuration
+type Threshold struct {
```

Comment:
ThresholdHealth sounds better description what represents I think.
Threshold is too generic.